keras_cv/layers/preprocessing/base_image_augmentation_layer_test.py

```diff
@@ -161,14 +151,22 @@ def test_augment_image_and_localization_data(self):
                 "segmentation_masks": segmentation_mask,
             }
         )
-
         expected_output = {
             "images": images + 2.0,
-            "bounding_boxes": bounding_boxes + 2.0,
+            "bounding_boxes": tf.ragged.constant(bounding_boxes + 2.0),
             "keypoints": keypoints + 2.0,
             "segmentation_masks": segmentation_mask + 2.0,
         }
-        self.assertAllClose(output, expected_output)
+
+        self.assertAllClose(output["images"], expected_output["images"])
+        self.assertAllClose(output["keypoints"], expected_output["keypoints"])
```

Comment:
lets add a generic `with_ragged_images_test.py` for all of our KPLs.  This can include all in RandAugment, etc